---
{"dg-publish":true,"permalink":"/evolutionary-biology/phylogenetics/"}
---

2025-11-17 - Dr. Peter Poczai

Not on exam - this is just for fun

Museonomics - genomes of plant herbarium specimens

# [[Evolutionary Biology/Taxonomy\|Taxonomy]]

## [[Herbariomics\|Herbariomics]]
- If we have a digitized collection, this gives us a lot of data other than just DNA - we collect all the metadata
- If we have collections going back to the 16th century, we can track evolutionary changes - windows to the past
# Answering questions related to Biodiversity Loss
- We can examine changes in the species based on the five causes of biodiversity loss
# CITES conention vs ICUN
- CITES - trade of endangered species - classifies species that are being traded or made money off of
- ICUN classifies species threatened by any other category (anthropogenic or otherwise)
# Example 1: Complicated data in *Solanum*
- In many cases, getting more genetic data does not resolve taxonomic problems - it can make things more complicated
- If they hybridize (reticulate evoution), different parts of the genome can be inherited in different ways between mitochondria
- In most cases, the phylogenetic tree will be describing evolutionary relationships - however, in plants, it might be less like a tree and more like a tangled web due to horizontal gene transfer
- "ghost taxa" - taxa that exist in the record but notclear
# Example 2 - plant invasions
- Common ragweed (*Ambrosia artemisiifolia*) has spread throughout Europe
	- Originally from north america (Sonoran desert) - arrived in grain shipments to Europe during the world wars
	- Maintains germination potential for 50 years - surfaced out of the seed bank through disturbances
	- Only plant in Asteraceae with a certain organ to shoot out pollen very far - can even send it as far as Finland
	- 2/3 of people are allergic to this plant. There is basically no way to get rid of this - certain biological controls like Ambrosia beetle and rust might work
- Why is this a "good" invasive species? What makes it invasive?
First, determine where it is coming from. Sampled historic and modern samples from North America and Europe. 

Something must be wrong - the modern European species have greater genetic diversity than would be expected based on the [[Evolutionary Biology/Population Genetics - Genetic Drift#Founder Effect\|founder effect]]

Theremust have been some introgression from another species - new admixture. Also there was DNA damage. We are also getting the bacterial community (meta genome) of this sample, from this we can infer the bacterial community that appeared with this plant

There are two native Ambrosias to european populations and two other invaders. Based on the microbial communities, we can see that the adaptive parts came from introgression from these european species. 

- Based on the [[Red Queen Hypothesis\|Red Queen Hypothesis]], some of the newly invaded populations, this species can spread because it lacks diseases. However, this was not true for Ambrosias, there were already existing diseases in Europe that could affect it, but it got these genes from the European species and created these hybrids. ==How does the microbial community fit in?==
- It also has glyphosate resistance

# Plant diseases from the past
- Based on historical samples
- *Rubus arcticus* disease which came from a host jumping - based on bacterial communities from the herbarium samples
# Cytonuclear Dissonance
- White-fruiting wild tomatoes and potatoes from the Andes
- In 2017 - we had 13 wild species of tomato including *Solanum habrochaites*, the domestic tomato
- When you compare the nuclear part and the plastid part, there are some changes (discordant) nodes between the two
- The interesting part is that *S. habrochaites* which is dispersed south to north in the Andes, but the populations interact with each other. They divide into two different parts. 
	- One plastome type is the red fruited kind and the other is green-fruited
	- If you are looking at genetic resistance to disease, we would want to cross our cultivated tomato with a species like *S. peruvianum*, but this does not work
	- The wild ancestor of cultivated tomatoes evolved at higher elevations and came down
	- In the huancobamba depression, this is the zone where *S. arcanum* and *S. habrochaites* grow in the same location but do not hybridize
	- However *S. habrochaites* and *S. pimpinellofolium* do hybridize in this area. Then they can back-cross with another species and end up with the plastomes "swapped" - this is wht we have these mismatches
	- If you have a couple SNPs in the plastome with a counterpart in the nucleus that match, the pollen tube goes straight down, otherwise the pollen tubes abort and the species cannot reproduce
	- Turns out the cultivated tomato is already a hybrid that cannot cross anymore with the wild species. It went through a genetic bottleneck in central america where it was domesticated by people. During domestication you have [[Selective Sweep\|Selective Sweep]]s that further reduce genetic diversity - now we have this domestic cultivated species with quite low genetic diversity
## Potato
- Turns out that thisis the result of a hybrid between the domestic tomato and the wild relative of potatoes. Tuber formation only began once it hybridized with domestic cultivated tomato. Now, after this hybridization, we have explosive diversity of tuber types in bursts - rapid diversification.
	- Molecular diversification accelerates and new traits appear through hybrid formation
	- Is this a new species? It certain doesn't fit this monophyletic phylogenetic evolutionary model
		- This only works under a rank-free clade where one species can have paraphyletic ancestors. This can be a hybridization network - a 
- Read more here [Tomato and Potato hybrids](https://www.cell.com/cell/fulltext/S0092-8674(25)00736-6)
# Part 2 - Extinct and Extant Taxa

## Heredity and Genetics
- Heredity ([[heritability\|heritability]]) is the same as genetics - for heredity think Mendel.
# Solanaceae - nightshade family
- Contains many economically and culturally important species, but also many kinds of adaptations and explosions of diversity. 
- These diversity explosions have long puzzled taxonomists
- Calibrating a molecular clock from the genes - this does not match the fossil record.
	- Using Bayesian calibrations - putting fossil dates on different nodes
	- Based on seed fossils placed on the evolutionary tree - based on morphological characteristics only
	- Errors are sometimes incorporated in the estimation based 
	- Wilf et al 2017 identified some *Physalis* fossils dated at 52 MYA. However, based on our molecular clock, these should be older than we have identified for the entire nightshade family
	- Then we found other fossils.
		- Theoretically, *Physalic infinemundi* which is dated to 52.2 Ma. How have we overlooked this?
		- Looking at the seeds in the fossil record - we probably misplaced these
		- Needed to revise the entire fossil record. Based on this fossil record, this changes the entire biogeographical history of this species
- Needed to refine the seed classification - used micro CT scans and PCA analysis to better analyze these. Then Bayesian models are used to classify. ==What exactly is going into the PCA? Measurements?==
	- Multidimensional scaling analysis - 14 morphological correctors are scaled from the measurements
		- Area, perimeter, roundess of the testal cells of seeds. This data needs to be transformed to put into an evolutionary model, in this case they classified them into 14 categories
## Tip dating vs node dating?
- Tip dating is more often used in pandemics. 
- They ysed a Bayesian Total Evidence Dating approach which simaultaneously infers topology and divergence traits, then averages them in Bayesian space. This allows the use of multiple fossils

This approach in fact inferred that the family emerged 98 Mya - much older than previous estimates. Matched up with other studies (using molecular information) that were discredited

Molecular clock calibration can be done in different ways - maximum likelihood distribution estimate - the oldest date in the Solanaceae evolutionary tree can be estimated with these other models (Huang et al. 2023) - more nodules at younger nodes.

Looking for a master's student in leaf trait evolution in Solanaceae.

# Ancient DNA (aDNA) vs Historical DNA (hDNA)
- Ancient DNA comes from fossils that contain small traces of DNA
- Historical DNA is on a shorter timescale and comes from herbaria samples or ice cores for example
# DNA Damage Detection
- Before working with fragmented DNA, we need to treat it
- A C - T change caused by DNA damage can distort results
- mapDamage computational framework for Python and R that quantifies DNA change patterns
	- Infers overhangs and damage patterns and the probabilities of those
	- Based on a reference genome - creates reference file. This is going to be missing parts, but the probability can be calculated by the program
	- Creates a theta distribution (mean difference rate between reference and sequenced sample)
	- Creates a rescaled BAM file to use in further research
- Another method is an enzymatic treatment - a USER enzyme can "reverse" the damage done
	- However, we will still have missing parts
## Example - Rye from an old couch
- This was found to be a "mammoth Rye" variety that is now extinct
- Based on the [[Selective Sweep\|Selective Sweep]] they identified in the genome, they found what it was related to and high amounts of polymorphism
	- Selective sweep usually reduces polymorphism (sweeps it away)
	- It's highly likely that this came from a certain rye breeder
	- There is a similar Finnish rye variety that is similar - they gave advice to make crosses with this
# Herbarium Contamination
- When you have fragmented DNA, there could be contamination with other species from the herbarium- like introduced bacteria, viruses or fungi
	- Tobacco Mosaic Virus could be contaminated in the herbarium from a lab tech going for a smoke
	- Human, Mouse, Rat DNA are often present as well
- Once these common contaminants have their genomes sequenced, you can run your sample through a decontamination pipeline to eliminate these genomes.
- Another method - kmer?? 
-
# Fifth method - IQTree
https://iqtree.github.io/
This approcach is basd on a machine learning model which iterates over many possibilities to create one true tree
- Want to feed the best gene trees

# 6 - assessing discordance among genes
- As in the tomato example where there is discordance between gene histories which distort the true phylogenetic history of the organism
- First you could use a so-called "concordance factor" (maximum likelihood
## Gene Concordance Factor gCF
- If your sequenced genes aren't long enough you might have conflicting signals in your topologies

## Site concordance Factor sCF
- How many mutations in that gene still support that node?

# PhyParts / PieCharts
- Sequencing 250 genes from the data and we can plot the proportions of how many genes agree with each other from that node
- Genes that disagree